ui/app/templates/workspace/projects/project/apps.hbs

```diff
@@ -3,6 +3,7 @@
     <h1>{{@model.name}}</h1>
     <small>{{pluralize @model.applicationsList.length "application"}}</small>
   </div>
+  <LinkTo @route="workspace.projects.project.settings" @model={{@model}}><Pds::Icon @type="settings"/> Settings </LinkTo>
```

Comment:
Just making sure you want to pass the whole model here and not the `id`, it looks like you're not fetching the model here: https://github.com/hashicorp/waypoint/pull/1057/files#diff-3741bb61d2053798b179535df28c0a731f67c2856d63f582e01cb90dda79ed6cR25 
but passing a whole object in a link will bypass route hooks so that may be an unwelcome surprise later !

Reply:
And now that I wrote that, I'm not sure if that's still true for routes that don't have params (which I realized is true for this route) 🤔